saleor/graphql/discount/types/vouchers.py

```diff
@@ -51,17 +61,31 @@ def resolve_channel(root: models.VoucherChannelListing, info: ResolveInfo):
 
 
 class Voucher(ChannelContextTypeWithMetadata[models.Voucher]):
-    id = graphene.GlobalID(required=True)
-    name = graphene.String()
-    code = graphene.String(required=True)
-    usage_limit = graphene.Int()
-    used = graphene.Int(required=True)
-    start_date = graphene.DateTime(required=True)
-    end_date = graphene.DateTime()
-    apply_once_per_order = graphene.Boolean(required=True)
-    apply_once_per_customer = graphene.Boolean(required=True)
-    only_for_staff = graphene.Boolean(required=True)
-    min_checkout_items_quantity = graphene.Int()
+    id = graphene.GlobalID(required=True, description="The ID of the voucher.")
+    name = graphene.String(description="The name of the voucher.")
+    code = graphene.String(required=True, description="The code of the voucher.")
+    usage_limit = graphene.Int(description="The number of times a voucher can be used.")
+    used = graphene.Int(
+        required=True, description="Determine if the voucher has been used."
```

Comment:
The field stores the number of voucher usage, so you can determine if the voucher has been used but also you can see how many times it was used.
I would change the description to something like:
`"The number of times a voucher has been used."`

Reply:
Usage count of the voucher.